src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+![Color anatomy](images/color_anatomy_bkg.svg)
+
+### Color terms
+
+| Term      | Definition                                                                                                  |
+| --------- | ----------------------------------------------------------------------------------------------------------- |
+| Theme | The set of unique values assigned to each token.                                                            |
+| Token | The code identifier for a unique role or set of roles. Tokens are universal and never change across themes. |
+| Role | The systematic usage/s of a value. Roles cannot be changed between themes.                                  |
+| Value | The actual style (ie hex code) assigned to a token.                         |
+
+
+## UI Color themes
+
+Our palette is designed and implemented in a themable manner. Each theme works with a specific set of primary background colors. These colors are global variables which determine the color of the component being applied within each theme. The light theme uses White and Gray 10 backgrounds and the dark theme uses Gray 100, 90 and 80 backgrounds. Default color tokens are provided for each component based on the primary background color.
+
+
+
+| Theme | Primary Background       | Token                | Hex value    |    |
+| --- | ----------- | ------------------- | -------------------- | ------------------- |
+| White   | Global Background Light  | `$ui-background`        | <color-block showhex="true" size="xs">#ffffff</color-block>  |
+| Gray 10   | Global Background Light | `$ui-background`  | <color-block showhex="true" size="xs">#f3f3f3</color-block>  |
+| Gray 90  | Global Background Dark   | `$ui-background` | <color-block showhex="true" size="xs">#282828</color-block>    |
+| Gray 100   | Global Background Dark  | `$ui-background`        | <color-block showhex="true" size="xs">#171717</color-block>     |
+
+
+
+
+### Tokens
+
+These themes are implemented with color tokens. In order to tokenize, a set of universal color variables must be determined by common roles and usage; tokens are not based singularly on a color value (i.e. unique hex code). The same color value may be assigned to multiple variables in a theme’s palette when the values have distinctly different roles.
+
+A universal variable can also have multiple associated roles when the color is consistently used across those roles. This allows for uniform color application across themes while giving each theme the freedom to express its own individuality at a more detailed level.  
+
+With this system, non-IBM users can also create their own theme using our color tokens.
+
+### Light theme
+
+Gray 10 components are generally paired with White backgrounds and White components with Gray 10 backgrounds. Some components, such as buttons and toggles, are common across both backgrounds. 
```

Comment:
Edits: 
Gray 10 components are paired with White backgrounds **to make the White Theme** and White components with Gray 10 backgrounds **to make the Gray 10 Theme**.  Some components, such as buttons and toggles, **share the same colors across both light themes**.

That last sentence isn't true with the way themeing is being implemented.

Reply:
no need to capitalize the names of colors here

Reply:
I still think we should update this paragraph. We need to make it clear that there are two distinct themes under light.